ext/node/ops/http.rs

```diff
@@ -133,89 +213,33 @@ where
     request.headers_mut().insert(CONTENT_LENGTH, len.into());
   }
 
-  let cancel_handle = CancelHandle::new_rc();
-  let cancel_handle_ = cancel_handle.clone();
-
-  let fut = async move {
-    client
-      .send(request)
-      .map_err(Into::into)
-      .or_cancel(cancel_handle_)
-      .await
-  };
-
-  let request_rid = state.resource_table.add(FetchRequestResource {
-    future: Box::pin(fut),
-    url,
-  });
-
-  let cancel_handle_rid =
-    state.resource_table.add(FetchCancelHandle(cancel_handle));
-
-  Ok(FetchReturn {
-    request_rid,
-    cancel_handle_rid: Some(cancel_handle_rid),
-  })
-}
+  let res = sender.send_request(request).map_err(Error::from).boxed();
+  let rid = state
```

Comment:
Is this on purpose that we don't allow to cancel the request now (ie. the `CancelHandle` is not created and used here)?

Reply:
I guess it's not on purpose. @satyarohith What do you think?

I'll try to restore it

Reply:
It's not on purpose.

 👍 to restore it.

Reply:
Restored the cancel handle in https://github.com/denoland/deno/pull/25470/commits/0e3e450c28414feb02e0ae4b58b700e15907d2c6